source/ui/dataman/AMSpectrumAndPeriodicTableView.h

```diff
@@ -156,6 +156,8 @@ protected slots:
 
 	/// The export button.
 	QPushButton *exportButton_;
+	///
```

Comment:
Also, you need to be careful using virtual functions in constructors (I'm not sure about using them in connect statements).  In general, the use of virtual functions in constructors should be avoided because the virtual function won't be properly resolved using polymorphism yet.